dxr/app.py

```diff
@@ -209,15 +209,18 @@ def browse(tree, path=''):
         return _browse_folder(tree, path.rstrip('/'), config)
```

Comment:
I believe we'll want to do this for folders as well: after all, a symlink's target can be a folder.